src/index.html

```diff
@@ -3,141 +3,173 @@
-            productive with Dart in just a few days.
-          </p>
+    <section class="row dash-row">
+        <div class="col-lg-7 col-md-6 col-sm-12 content-image"><img class="galleryTwo"
+                src="assets/dash/2x/3-1 hot reload iterative changes@2x-80.jpg"/>
+        </div>
+        <div class="col-lg-5 col-md-6 col-sm-12 content-info">
+            <div class="content-container content-container-left">
+                <div class="content-icon"></div>
+                <div class="content-feature">Client-Optimized  Language</div>
+                <div class="content-desc" id="galleryTwo">
+                    <ul class="selector">
+                        <li class="highlight"  data-banner="assets/dash/2x/3-1 hot reload iterative changes@2x-80.jpg">AOT-compile mobile and desktop apps to machine code for instant startup</li>
+                        <li data-banner="assets/dash/2x/3-2 runtime insights@2x-80.jpg">AOT-compile mobile and desktop apps to machine code for instant startup</li>
+                        <li data-banner="assets/dash/2x/3-3 rich tools@2x-80.jpg">AOT-compile mobile and desktop apps to machine code for instant startup</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="clearfix"></div>
+        </div>
+    </section>
 
-          <h2>Reactive</h2>
-          <p>
-            Dart is well-suited to reactive programming, with support for
-            managing short-lived objects—such as UI widgets—through
-            Dart’s fast object allocation and
-	    <a href="https://medium.com/flutter-io/flutter-dont-fear-the-garbage-collector-d69b3ff1ca30">generational garbage collector.</a>
-            Dart supports asynchronous programming through
-            language features and APIs that use
-            <a href="/guides/libraries/library-tour#future">Future</a> and
-            <a href="/guides/libraries/library-tour#stream">Stream</a> objects.
-          </p>
+    <section class="row dash-row">
+        <div class="col-lg-5 col-md-6 col-sm-12 content-info">
+            <div class="content-container">
+                <div class="content-icon"></div>
+                <div class="content-feature">Productive  Development</div>
+                <div class="content-desc" id="galleryThree">
+                    <ul class="selector">
+                        <li class="highlight"  data-banner="assets/dash/2x/3-1 hot reload iterative changes@2x-80.jpg">AOT-compile mobile and desktop apps to machine code for instant startup</li>
+                        <li data-banner="assets/dash/2x/3-2 runtime insights@2x-80.jpg">AOT-compile mobile and desktop apps to machine code for instant startup</li>
+                        <li data-banner="assets/dash/2x/3-3 rich tools@2x-80.jpg">AOT-compile mobile and desktop apps to machine code for instant startup</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="clearfix"></div>
```

Comment:
I could be missing something but I don't see a reason to use these. Consider taking them out if we can

Reply:
The clearfix is there because content-container has a float:right or float:left (since the position of the blocks alternate)